Habitica/src/com/habitrpg/android/habitica/MainActivity.java

```diff
@@ -739,4 +745,31 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
         return super.onKeyUp(keyCode, event);
     }
+
+    private void scheduleReminder(Context context) {
+
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        String timeval = prefs.getString("reminder_time", "19:00");
+
+        if (timeval == null) timeval = "19:00";
```

Comment:
Fixed this and also fixed the crashes where tiemval was `null`. The default values of the preferences were never set.